packages/search-in-workspace/package.json

```diff
@@ -9,7 +9,6 @@
     "@theia/navigator": "^1.7.0",
     "@theia/process": "^1.7.0",
     "@theia/workspace": "^1.7.0",
-    "minimatch": "^3.0.4",
```

Comment:
I believe you made this change by mistake when rebasing. Please revert it as `minimatch` is required for siw.